packages/design-system/src/components/web-components/ds-alert.test.tsx

```diff
@@ -0,0 +1,180 @@
+import React from 'react';
+import Alert, { AlertProps } from '../Alert/Alert';
```

Comment:
This doesn't actually import the `ds-alert.ts` module, so I'm not sure what it's doing when it renders. I tried swapping this out for `import './ds-alert';`, removing the `register` call (because `ds-alert.ts` does it), and temporarily simplifying the types that complain to `any`, and when it takes the snapshots, it has different results. I think these four modules need to switch to importing from their respective web components modules. Upside is that I don't think we'll need `preact-custom-element`.